apps/web/src/pages/TalentNominations/NominationGroup/CareerExperience.tsx

```diff
@@ -180,6 +261,10 @@ const TalentNominationGroupCareerExperience_Query = graphql(/* GraphQL */ `
       nominee {
```

Comment:
You're querying the same fields twice?

Reply:
ahh.. sorry .. I had a bad merge conflict.. will remove it

Reply:
Don't forget this fix.  :grin: